Client/Frontend/Library/LibraryViewController/LibraryViewController+LibraryPanelDelegate.swift

```diff
@@ -6,38 +6,15 @@ import Storage
 import Shared
 
 extension LibraryViewController: LibraryPanelDelegate {
-    func libraryPanelDidRequestToSignIn() {
-        self.dismiss(animated: false, completion: nil)
-        delegate?.libraryPanelDidRequestToSignIn()
-    }
-
-    func libraryPanelDidRequestToCreateAccount() {
-        self.dismiss(animated: false, completion: nil)
-        delegate?.libraryPanelDidRequestToCreateAccount()
-    }
-
     func libraryPanelDidRequestToOpenInNewTab(_ url: URL, isPrivate: Bool) {
         delegate?.libraryPanelDidRequestToOpenInNewTab(url, isPrivate: isPrivate)
+        DispatchQueue.main.async {
+            self.dismiss(animated: true, completion: nil)
+        }
```

Comment:
Why do we need this dismiss here?

Reply:
Oh forgot about that. So there's actually a bug inside the app, if you tap and hold on a link in library and choose to open in new tap the library never dismisses.

Reply:
The `DispatchQueue` is added because there's an overlay presented to select open in new tab and it has to wait somehow for it to be dismissed. It happens a lot with the Settings menu.